src/js/components/Grid/StyledGrid.js

```diff
@@ -4,19 +4,19 @@ import { borderStyle, edgeStyle, genericStyles } from '../../utils';
 import { defaultProps } from '../../default-props';
 
 const fillStyle = fill => {
+  if (fill === false || fill === undefined) {
+    return false;
```

Comment:
This function used to return `undefined` for 'false' or 'undefined' values. Any reason it is now returning 'false'?